common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -282,12 +326,15 @@
             "error"
         ],
         "@typescript-eslint/restrict-template-expressions": [
-            "off"
+            "warn"
         ],
         "@typescript-eslint/semi": [
             "error",
             "always"
         ],
+        "@typescript-eslint/sort-type-union-intersection-members": [
+            "warn"
+        ],
```

Comment:
This one has an auto-fixer, so even if we leave it as a warning, it's going to end up changing a bunch of code. How do folks feel about that?

Reply:
Prob leave off then and consider it in isolation.

Reply:
Done in latest.